src/workerd/server/container-client.c++

```diff
@@ -335,6 +340,25 @@ kj::Promise<void> ContainerClient::killContainer(uint32_t signal) {
       "Stopping container failed with: ", response.body);
 }
 
+// This method assumes that the container is already running.
+kj::Promise<void> ContainerClient::destroyContainer() {
+  auto endpoint = kj::str("/containers/", containerName, "?force=true");
+  auto response = co_await dockerApiRequest(
+      network, kj::str(dockerPath), kj::HttpMethod::DELETE, kj::mv(endpoint));
+  // statusCode 204 refers to "no error"
+  // statusCode 404 refers to "no such container"
+  // Both of which are fine for us since we're tearing down the container anyway.
+  JSG_REQUIRE(response.statusCode == 204 || response.statusCode == 404, Error,
+      "Removing a container failed with: ", response.body);
+  {
```

Comment:
question for my understanding: why do you create a new scope here?

Reply:
No need. I don't remember why I did that in the first place. Probably code evolved so that this is not needed anymore. 

I'll remove it.